src/Stripe.net/Entities/AccountLinks/AccountLink.cs

```diff
@@ -17,14 +17,14 @@ public class AccountLink : StripeEntity<AccountLink>, IHasObject
         /// </summary>
         [JsonProperty("created")]
         [JsonConverter(typeof(UnixDateTimeConverter))]
-        public DateTime Created { get; set; }
+        public DateTime Created { get; set; } = new DateTime(1970, 1, 1, 0, 0, 0);
```

Comment:
We should not pass a value directly. Instead, we should define our own constant somewhere, e.g. in a `Stripe.Infrastructure.DateTimeUtils` class:
```c#
public static readonly DateTime UnixEpoch = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
```

then initialize values with `= Stripe.Infrastructure.DateTimeUtils.UnixEpoch;`

(Also maybe a leave a comment to just use `DateTime.UnixEpoch` once we bump the minimum supported version to .NET Standard 2.1`)